app/handlers.go

```diff
@@ -0,0 +1,66 @@
+package app
+
+import (
+	"GOBootcamp/app/models"
+	"encoding/csv"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+func (a *App) IndexHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, "Welcome to POST API")
+	}
+}
+
+func (a *App) GetPostHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+
+		f, err := os.Open("test.csv")
+		if err != nil {
+			log.Println("Unable to read input file test.csv", err)
+			sendResponse(w, r, nil, http.StatusInternalServerError)
+			return
+		}
+
+		defer func(f *os.File) {
+			err := f.Close()
+			if err != nil {
+				log.Println("Unable to close file as CSV for ../../test.csv", err)
+			}
+		}(f)
+
+		csvReader := csv.NewReader(f)
```

Comment:
In this function, it would be difficult later to have each layer separated, whenever you want we could have a session to explain the different layers, but keep in mind that this architecture should look like this in the basics:
Router -> call handler -> call service -> call repository